mis_builder/models/mis_report_instance.py

```diff
@@ -135,6 +135,16 @@ def _compute_dates(self):
                 record.date_from = fields.Date.to_string(date_from)
                 record.date_to = fields.Date.to_string(date_to)
                 record.valid = True
+            elif record.mode == MODE_REL and record.type == 'ytd':
+                date_from = d.replace(month=1, day=1)
+                date_from = date_from + \
+                    relativedelta(years=record.offset)
+                today = fields.Date.context_today(record)
+                date_to = fields.Date.from_string(today) + \
+                    relativedelta(years=record.offset)
```

Comment:
@MaxyMoos I've a little doubt about this. Let's say you have a base date of '2018-05-05' and offset 0.
This would give you a period of '2018-01-01' to '2019-05-05'. That might be unexpected.

Also, I'm wondering if we don't need variants such as 'year to end of month', etc. For accounting reports for instance, `today` is rarely a very useful date.

So maybe this ytd option should instead be a flag next to the 'mode' and `type` fields where `date_to` is computed with the current algorithm and `date_start` replaces month and day by 1 in date_start.

Reply:
@sbidoul Indeed that sounds like a much better approach. I pushed such modifications with some tests just now. If it looks good to you I'll scrap the previous commit to keep a clean history in the repo.